.github/workflows/ci-wordpress.yaml

```diff
@@ -0,0 +1,98 @@
+name: CI-Wordpress
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for deploy wordpress to shared hosting by ansistrano ansible role
+#
+# Original Author: Rostislav Udaltsov
+# Created: 09/19/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      server_host:
+        required: true
+        type: string
+        description: 'The address of the server'
+      environment:
+        required: true
+        type: string
+        description: 'The environment'
+      playbook_path:
+        required: true
+        type: string
+        description: 'The path to the file relative to the repository'
+      deploy_path:
+        required: true
+        type: string
+        description: 'The path to deploy on the production server'
+      deploy_repo:
+        required: true
+        type: string
+        description: 'The repository for deploy'
+      deploy_branch:
+        required: true
+        type: string
+        description: 'The branche for deploy'
+      python_version:
+        required: true
+        type: number
+        description: 'The version of python'
+      runner_image:
+        required: true
+        type: string
+        description: 'The image of the runner OS on which workflow is running'
+    # Secrets for ssh
+    secrets:
+      ssh_deploy_user:
+        required: true
+        description: 'The username for connect to production server via ssh'
+      ssh_private_key:
+        required: true
+        description: 'The private key for connect to production server via ssh'
+
+jobs:
+  deploy:
+    runs-on: ${{ inputs.runner_image }}
+
+    steps:
+      # Preparing workspace: install python, ansible and ansistrano-role
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
```

Comment:
Why is action version so low? Currently there is [v4.7](https://github.com/marketplace/actions/setup-python?version=v4.7.0) available. I would try the latest version and see if it works.

Reply:
updated